drivers/interrupt_controller/intc_gd32_exti.c

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
Note: I'm not sure if implementing EXTI as a sort of "interrupt controller" is in the end a good idea. EXTI isn't really an interrupt controller on its own (NVIC is used), it's just a set of registers that allow to configure multiple external interrupt sources, being GPIO one of them. I'll try embedding this directly into GPIO and compare the result.

Reply:
SAM0 uses NVIC line too. What is important know is EXTI depends on GPIO ? I mean, EXTI only works from lines [0-15] when GPIO clock is enabled?

Reply:
Dropping the idea for now, I think it's acceptable as it is now.